### PR TITLE
Final changes for v1.2.1

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
1.2.1 ready to release

### CODE REVIEW

File: scripts/gen-descriptions.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import sys
 import xml.dom.minidom as md
 
-title_tmpl = u"{title-ios}"
+title_tmpl = u"{title-android}"
```
Comment (bassosimone) [inline scripts/gen-descriptions.py]: Is this correct?

Comment (lorenzoPrimi) [reply scripts/gen-descriptions.py]: Yes, now iOS only allows 30 char in the title. Will be renamed "title" soon

**Comment by bassosimone** [comment]
I've restarted the build that failed for a transient error after the last commit

**Comment by bassosimone** [comment]
@lorenzoPrimi can you make another test flight from this updated branch?

File: ooniprobe/AppDelegate.mm [hunk 2]
```diff
@@ -122,7 +121,7 @@ - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)butto
     }
     else if (buttonIndex == 1 && alertView.tag == alert_tag_open_itunes){
         //open ooniprobe on iTunes connect
-        NSString *iTunesLink = @"itms://itunes.apple.com/us/app/apple-store/id1245670385?mt=8";
+        NSString *iTunesLink = @"itms://itunes.apple.com/us/app/apple-store/id1199566366?mt=8";
```
Comment (hellais) [inline ooniprobe/AppDelegate.mm]: What is the `mt=8` at the end for?

File: ooniprobe/Model/NetworkMeasurement.mm [hunk 3]
```diff
@@ -16,11 +12,17 @@
 #include <resolv.h>
 #include <dns.h>
 
+#define verbosity MK_LOG_INFO
+#define anomaly_green 0
+#define anomaly_orange 1
+#define anomaly_red 2
```
Comment (hellais) [inline ooniprobe/Model/NetworkMeasurement.mm]: It's generally common practice to define constants using `CAPTAIN_CAPS_LOCK_CASE`

File: ooniprobe/AppDelegate.mm (hunk 2 above)
Comment (lorenzoPrimi) [reply ooniprobe/AppDelegate.mm]: https://stackoverflow.com/questions/1781427/what-is-mt-8-in-itunes-links-for-the-appstore

**Comment by hellais** [comment]
nuke [2017-09-27 12:47 PM] 
iOS (31) sent to Apple

**Pull request merged**